lib/location/loc_utils.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef LOC_UTILS_H
+#define LOC_UTILS_H
+
+struct loc_utils_modem_params_info {
+	/** Mobile Country Code. */
+	int mcc;
+
+	/** Mobile Network Code. */
+	int mnc;
+
+	/** E-UTRAN cell ID */
+	uint32_t cell_id;
```

Comment:
Looking at loc_utils_modem_params_read(), it seems that 16 bit would be enough? Same goes for TAC

Reply:
lte_lc has `cell_id` and `tac` as `uint32_t` so we should have the same. `phys_cell_id` is 16 bits and we have separate variable for that.
@hiltunent Please close if this responds to your questions.